src/utils.rs

```diff
@@ -27,6 +32,24 @@ pub fn extract_signature_components(sig: &Signature) -> (u8, B256, B256) {
     (v, r, s)
 }
 
+/// Populates the initial gas parameters for a transaction request based on the
+/// previous block header.
+pub fn populate_initial_gas(req: &mut TransactionRequest, prev_header: &Header) {
+    const STARTING_MPFPG: u128 = 2 * GWEI_TO_WEI as u128;
+
+    let base_fee_per_gas = prev_header
+        .next_block_base_fee(BaseFeeParams::ethereum())
+        .expect("signet deployed after 1559 active") as u128;
+
+    let blob_basefee = prev_header
+        .next_block_blob_fee(BlobParams::prague())
+        .expect("signet deployed after 4844 active");
```

Comment:
So, this `expect` was panicking at runtime. I updated it to an `unwrap_or` and it defaults to the Prague value, but I'm not sure _why_ it was panicking.